roles/create_certification_project/tasks/validate_cert_settings.yml

```
@@ -15,7 +15,30 @@
       - "cert_settings.repository_description is defined"
     fail_msg: "The parameter {{ item }} is required to update cert project"
     success_msg: "The parameter {{ item }} is present"
-  when: product_type == 'container'
+  when:
+    - product_type == 'container'
+    - do_container_parallel_test
```

Can we add a condition to this line to make it clear? something like:
```suggestion
    - do_container_parallel_test | default(False) | bool
```